Fauxton failing to build on Centos 7 CI container

#Expected Behavior
`npm run webpack:release` should run successfully on all supported platforms.

## Current Behavior
Fauxton is failing to build on the Centos 7 CI container (`couchdbdev/centos-7-erlang-19.3.6:latest`). Currently the container uses:

 * npm 6.4.1
 * node v8.16.0

Error is:

```
ERROR in ./app/main.js
Module build failed (from ./node_modules/eslint-loader/index.js):
Error: Cannot find module 'acorn'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (/usr/src/couchdb/src/fauxton/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/usr/src/couchdb/src/fauxton/node_modules/acorn-jsx/index.js:8:15)
    at Module._compile (/usr/src/couchdb/src/fauxton/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (/usr/src/couchdb/src/fauxton/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/usr/src/couchdb/src/fauxton/node_modules/espree/espree.js:62:13)
    at Module._compile (/usr/src/couchdb/src/fauxton/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (/usr/src/couchdb/src/fauxton/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/usr/src/couchdb/src/fauxton/node_modules/eslint/lib/linter/linter.js:17:14)
    at Module._compile (/usr/src/couchdb/src/fauxton/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (/usr/src/couchdb/src/fauxton/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/usr/src/couchdb/src/fauxton/node_modules/eslint/lib/linter/index.js:3:20)
    at Module._compile (/usr/src/couchdb/src/fauxton/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (/usr/src/couchdb/src/fauxton/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/usr/src/couchdb/src/fauxton/node_modules/eslint/lib/cli-engine/cli-engine.js:24:20)
    at Module._compile (/usr/src/couchdb/src/fauxton/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
 @ multi core-js/fn/array core-js/fn/string/ends-with core-js/fn/string/starts-with core-js/fn/object core-js/fn/symbol core-js/fn/promise regenerator-runtime/runtime ./app/main.js bundle[7]
```

Fauxton 1.2.0 updated webpack from `"~3.12.0` to `^4.35.3` so I wonder whether https://github.com/webpack/webpack/issues/8656 is related. I cannot reproduce the problem locally on MacOS with the same node/npm versions but can produce the error in the Docker container.

## Possible Solution
 * Add webpack:release to the fauxton build / Travis so we verify the step there
 * Add an explicit dependency on acorn?

## Steps to Reproduce (for bugs)
A simple Dockerfile to reproduce the error:

```
FROM couchdbdev/centos-7-erlang-19.3.6:latest
USER root
WORKDIR /usr/src

RUN set -xe; \
    git clone https://github.com/apache/couchdb-fauxton; \
    cd couchdb-fauxton; \
    npm install --production && ./node_modules/grunt-cli/bin/grunt couchdb
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,12 +1,11 @@
 language: node_js
 node_js:
-   - "8.9"
+   - "10"
 sudo: false
 services:
   - docker
 git:
   depth: 1
-
 env:
   - COUCHDB_IMAGE=apache/couchdb:2.3.1 NIGHTWATCH_SKIPTAGS="search,partitioned"
   - COUCHDB_IMAGE=couchdb:dev NIGHTWATCH_SKIPTAGS="search,nonpartitioned"
@@ -21,6 +20,7 @@ before_script:
   - travis_retry ./bin/docker-up-and-check.sh
   - curl http://127.0.0.1:5984
   - npm run stylecheck
+  - npm ls --production --depth=0
   - npm test
   - grunt debugDev
   - DIST=./dist/debug ./bin/fauxton &
@@ -29,7 +29,6 @@ script:
   - ./node_modules/.bin/grunt nightwatch_retries
 after_script:
   - npm run docker:down
-
 cache:
   - apt
   - directories:

--- a/app/addons/documents/rev-browser/actions.js
+++ b/app/addons/documents/rev-browser/actions.js
@@ -13,7 +13,7 @@
 import FauxtonAPI from "../../../core/api";
 import {post} from "../../../core/ajax";
 import ActionTypes from "./actiontypes";
-import getTree from "visualizeRevTree/lib/getTree";
+import getTree from "visualize-rev-tree/lib/getTree";
 import PouchDB from "pouchdb-core";
 import PouchHttpAdapter from 'pouchdb-adapter-http';
 PouchDB.plugin(PouchHttpAdapter);

--- a/jest-setup.js
+++ b/jest-setup.js
@@ -14,7 +14,13 @@ require('jest');
 require('whatwg-fetch');
 require('mock-local-storage');
 
-window.localStorage = global.localStorage;
+Object.defineProperty(window, 'localStorage', {
+  value: global.localStorage,
+  configurable:true,
+  enumerable:true,
+  writable:true
+});
+
 window.$ = window.jQuery = require('jquery');
 window._ = require('lodash');
 window.Backbone = require('backbone');

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "fauxton",
-  "version": "1.2.1",
+  "version": "1.2.2",
   "description": "Fauxton is a modular CouchDB dashboard and Futon replacement.",
   "main": "./index.js",
   "directories": {
@@ -34,6 +34,8 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.4.4",
     "@webcomponents/url": "~0.7.1",
+    "acorn": "^6.0.0",
+    "ajv": "^6.9.1",
     "async": "~0.2.6",
     "babel-eslint": "^9.0.0",
     "babel-loader": "^8.0.6",

--- a/package.json
+++ b/package.json
@@ -104,7 +104,7 @@
     "url-loader": "^2.0.1",
     "urls": "~0.0.3",
     "uuid": "^3.0.1",
-    "visualizeRevTree": "git+https://github.com/neojski/visualizeRevTree.git#gh-pages",
+    "visualize-rev-tree": "0.0.1",
     "webpack": "^4.41.0",
     "webpack-cli": "^3.3.9",
     "webpack-dev-server": "^3.8.2",

